disconnected/about.adoc

```diff
@@ -0,0 +1,61 @@
+
+[id="glossary-disconnected_{context}"]
+== Glossary of disconnected environment terms
+
+Although it is used throughout the {product-title} documentation, _disconnected environment_ is a broad term that can refer to environments with various levels of internet connectivity.
+Other terms are sometimes used to refer to a specific level of internet connectivity, and these environments might require additional unique configurations.
+
+The following table describes the different terms used to refer to environments without a full internet connection:
+
+.Disconnected environment terms
+[cols=".^2,.^4",options="header"]
+|====
+|Term |Description
+
+|Air-gapped network
+|An environment or network that is completely isolated from an external network.
+
+This isolation depends on a physical separation, or an "air gap", between machines on the internal network and any other part of an external network.
+Air-gapped environments are often used in industries with strict security or regulatory requirements.
+
+|Disconnected environment
+|An environment or network that has some level of isolation from an external network.
+
+This isolation could be enabled by physical or logical separation between machines on the internal network and an external network.
+Regardless of the level of isolation from the external network, a cluster in a disconnected environment does not have access to public services hosted by Red{nbsp}Hat and requires additional setup to maintain full cluster functionality.
+
+|Restricted Network
+|An environment or network with limited connection to an external network.
+
+A physical connection may exist between machines on the internal network and an external network, but network traffic is limited by additional configurations, such as with firewalls and proxies.
+|====
+
+[id="preferred-methods_{context}"]
+== Preferred methods for working with disconnected environments
+
+// Not sure about the heading above, I'm trying to find some phrasing that basically says "You should probably use these things if you don't have a strong reason to go with other options".
+
+You can choose between multiple options for most aspects of managing a cluster in a disconnected environment.
+For example, when mirroring images you can choose between using the oc-mirror {oc-first} plugin or using the `oc adm` command.
+
+However, some options provide a simpler and more convenient user experience for disconnected environments, and are the preferred method over their alternatives.
+
+Unless your organizational needs require you to choose another option, use the following methods for mirroring images, installing your cluster, and updating your cluster:
+
+* Mirror your images using the xref:../disconnected/mirroring/installing-mirroring-disconnected.adoc#installing-mirroring-disconnected[oc-mirror plugin].
```

Comment:
@skopacz1 Can you confirm the v2 GA and deprecation of oc adm release timelines? If they are targeted for 4.18, we can simplify this effort by simply highlighting the deprecation notices

Reply:
In the works but 4.18 is targeted. See https://github.com/openshift/oc-mirror/pull/946

Reply:
@jianlinliu oc-mirror v1 is currently GA and is the preferred choice, at least according to what I've heard. It's just that an improved v2 is set to GA in 4.18.

I think that when v2 becomes GA I will specify that here, otherwise I have to mention v1 for now

Reply:
Correct, oc-mirror v1 is supported but will be deprecated eventually, and we want oc-mirror v2 to be used instead once it GAs in 4.18.